### PR TITLE
feature/SKED-3718: Make sure that GET requests in the SDK can be retried when they fail

### PR DESCRIPTION
**Pull request by RobinMalfait**
> See JIRA: https://skedify.atlassian.net/browse/SKED-3718

### CODE REVIEW

**Comment by RobinMalfait** [comment]
Also added node support. We need to wait for https://github.com/skedify/babel-plugin-transform-polyfills/pull/22

**Comment by RobinMalfait** [comment]
BLOCKED BY skedify/babel-plugin-transform-polyfills#22

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "skedify-sdk",
   "version": "0.0.0-development",
-  "description": "Javascript SDK that helps integrators to interact with the Skedify API",
+  "description": "JavaScript SDK that helps integrators to interact with the Skedify API",
```
Comment (kendepelchin) [inline package.json]: 👍

File: README.md [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 ## Goal
 
-The goal of this SDK is to offer an easy to use tool to interact with the Skedify API when you're making a javascript integration.
+The goal of this SDK is to offer an easy to use tool to interact with the Skedify API when you're making a JavaScript integration.
```
Comment (kendepelchin) [inline README.md]: 👍

File: package.json [hunk 3]
```diff
@@ -83,6 +83,7 @@
   },
   "dependencies": {
     "axios": "^0.18.0",
-    "moxios": "^0.4.0"
+    "moxios": "^0.4.0",
+    "uuid": "^3.2.1"
```
Comment (kendepelchin) [inline package.json]: 👍

**Pull request merged**